ui/app/src/components/shared/notifications/NotificationPanel.tsx

```diff
@@ -1,159 +1,61 @@
+  const [calloutDetails, setCalloutDetails] = useState({ title: '', text: '', success: true });
   const apiCall = useAuthApiCall();
 
-  // NOTE: this first useEffect() hook will walk the tree of the TRE and add any running notifications to the panel on page load.
-  // It's very inefficient, and in future should be replaced by a single call to a new API Endpoint.
-  // For now, it's behind a feature flag. To turn off, see the config.json - loadAllOpsOnStart
   useEffect(() => {
-    const getOpsFromResourceList = async (resources: Array<Resource>, clientId?: string) => {
-
-      let opsToAdd: Array<Operation> = [];
-      const tasks: Array<any> = [];
-
-      resources.forEach(async (r: Resource) => {
-        tasks.push(apiCall(`${r.resourcePath}/${ApiEndpoint.Operations}`, HttpMethod.Get, clientId));
-      });
-      const results = await Promise.all(tasks);
-      results.forEach((r: any) => {
-        if (r && r.operations) {
-          r.operations.forEach((o: Operation) => {
-            if (inProgressStates.includes(o.status)) { opsToAdd.push(o) }
-          });
-        }
-      });
-      return opsToAdd;
-    };
-
     const loadAllOps = async () => {
-      console.warn("LOADING ALL OPERATIONS...");
-      let opsToAdd: Array<Operation> = [];
-      let sharedServiceList = (await apiCall(ApiEndpoint.SharedServices, HttpMethod.Get)).sharedServices as Array<Resource>;
-      sharedServiceList && sharedServiceList.length > 0 && (opsToAdd = opsToAdd.concat(await getOpsFromResourceList(sharedServiceList)));
-      let workspaceList = (await apiCall(ApiEndpoint.Workspaces, HttpMethod.Get)).workspaces as Array<Resource>;
-      workspaceList && workspaceList.length > 0 && (opsToAdd = opsToAdd.concat(await getOpsFromResourceList(workspaceList)));
-      for (let i=0;i<workspaceList.length;i++){
-        let w = workspaceList[i];
-        let appId = w.properties.scope_id;
-        if (w.properties.client_id === "auto_create") continue; // skip getting stuff from the workspace if it's auto_create as it doesn't have auth setup yet
-        let workspaceServicesList = (await apiCall(`${w.resourcePath}/${ApiEndpoint.WorkspaceServices}`, HttpMethod.Get, appId)).workspaceServices as Array<Resource>;
-        if (workspaceServicesList && workspaceServicesList.length > 0) (opsToAdd = opsToAdd.concat(await getOpsFromResourceList(workspaceServicesList, appId)));
-        for(let n=0;n<workspaceServicesList.length;n++){
-          let ws = workspaceServicesList[n];
-          let userResourcesList = (await apiCall(`${ws.resourcePath}/${ApiEndpoint.UserResources}`, HttpMethod.Get, appId)).userResources as Array<Resource>;
-          if (userResourcesList && userResourcesList.length > 0) (opsToAdd = opsToAdd.concat(await getOpsFromResourceList(userResourcesList, appId)));
-        }
-      }
+      let opsToAdd = (await apiCall(`${ApiEndpoint.Operations}/my`, HttpMethod.Get)).operations as Array<Operation>;
```

Comment:
Couldn't the `ApiEndpoint.Operations` incorporate the `my`?
Or, could the API not require the `my` at all?

I can't see many/any other instances where we've hard-coded part of the API path in the UI